metadata.json

```diff
@@ -46,33 +46,33 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "5.0",
-        "6.0",
-        "7.0"
+        "5",
+        "6",
+        "7"
       ]
     },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "5.0",
-        "6.0",
-        "7.0"
+        "5",
+        "6",
+        "7"
       ]
     },
     {
-      "operatingsystem": "Scientific Linux",
+      "operatingsystem": "Scientific",
```

Comment:
These `operatingsystem` changes surprised me. Also operatingsystemrelease. This matches what stdlib has, so I buy it.

Reply:
I checked facterdb, and it looks like correct name of the operatingsystem is just `Scientific` and not `Scientific Linux`